README.md

```diff
@@ -7,12 +7,14 @@ programming language with both textual and graphical syntaxes.
 
 for more info: http://ballerinalang.org/
 
-
 ## Installing the plugin to Visual Studio Code(vscode)
 
-1. Go to your home directory & locate .vscode/extentions/ folder
-2. place extracted zip content into that
-3. Open the vscode and edit desired bal files
+1. Clone the ballerina language server [repo](). Use `mvn clean install`. vscode plugin need the language server launcher executable.
```

Comment:
I don't think people have mvn pre-installed. shall we add steps to dependancy install

Reply:
Added instructions to install maven and nodejs.